### PR TITLE
Expand Featured Artists with osu! accounts

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/Featured_Artists/en.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ If a pre-timed `.osz` from the Featured Artist listing has errors of any kind, t
 
 ## Featured Artists with osu! accounts
 
-Featured Artists with osu! accounts are given the **Featured Artist** [title](/wiki/Community/User_title) and [user group](/wiki/People/User_group) to show their involvement. The [Featured Artist group page](https://osu.ppy.sh/groups/35) lists all such accounts, while the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) lists all Featured Artists.
+Featured Artists with osu! accounts are given the cyan [user title](/wiki/Community/User_title) and [user group](/wiki/People/User_group) badge that reads `FA` to show their involvement.
```
Comment (aceticke) [inline wiki/People/Featured_Artists/en.md:47]:
```suggestion
Featured Artists with osu! accounts are given the Featured Artist group which can be identified by it's cyan [user title](/wiki/Community/User_title) and [user group](/wiki/People/User_group) badge that reads `FA` to show their involvement.
```
better wording

**Review by aceticke — CHANGES_REQUESTED** [review]

File: wiki/People/Featured_Artists/en.md (hunk 1 above)
Comment (peppy) [reply wiki/People/Featured_Artists/en.md:47]: its

Comment (aceticke) [reply wiki/People/Featured_Artists/en.md:47]: fixed

File: wiki/People/Featured_Artists/en.md [hunk 2]
```diff
@@ -44,7 +44,9 @@ If a pre-timed `.osz` from the Featured Artist listing has errors of any kind, t
 
 ## Featured Artists with osu! accounts
 
-Featured Artists with osu! accounts are given the **Featured Artist** [title](/wiki/Community/User_title) and [user group](/wiki/People/User_group) to show their involvement. The [Featured Artist group page](https://osu.ppy.sh/groups/35) lists all such accounts, while the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) lists all Featured Artists.
+Featured Artists with osu! accounts are given the **Featured Artist** group, which can be identified by a cyan [user title](/wiki/Community/User_title) and [user group](/wiki/People/User_group) badge that reads `FA` to show their involvement.
```
Comment (Walavouchey) [inline wiki/People/Featured_Artists/en.md:47]: maybe better like this

```suggestion
Featured Artists with osu! account can be identified by a cyan "Featured Artist" [user title](/wiki/Community/User_title), and [user group](/wiki/People/User_group) badge that reads `FA` to show their involvement.
```

**Pull request merged**